docs/tutorial/kymotracking.rst

```diff
@@ -110,6 +116,55 @@ Fortunately, the signal to noise level in this kymograph is quite good. In pract
 one will have to resort to some fine tuning of the algorithm parameters over different regions of the kymograph to get
 an acceptable result.
 
+Gaussian Maximum Likelihood Estimation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The function `refine_lines_gaussian()` instead uses an MLE optimization of a Poisson likelihood with a Gaussian expectation
+to characterize both the expected peak shape and photonic noise of the observed signal, adapted from :cite:`mortensen2010gauloc`.
+A sub-sample of the image centered at the previously tracked peak for each time frame is fitted to the following likelihood function:
+
+.. math::
+
+    \mathcal{L} = \prod_i^M e^{-E_i} \frac{E_i^{n_i}}{n_i!}
+
+where :math:`M` is the number of pixels to be fit and :math:`n_i` and :math:`E_i` are the observed photon count and expectation value
+for pixel :math:`i`. The shape of the peak is described with a Gaussian expectation function
+
+.. math::
+
+    E_i = \frac{N a}{\sqrt{2 \pi \sigma^2}} \exp \left[ \frac{-(x_i-\mu)^2}{2 \sigma^2} \right] + b
+
+Here :math:`N` is the total photons emitted in the fitted image (line), :math:`a` is the pixel size, :math:`\mu` is the peak center,
+:math:`x_i` is the pixel center position, :math:`\sigma^2` is the distribution variance, and :math:`b` is the background level in
+photons/pixel.
```

Comment:
I would probably just call `sigma^2` just the variance for simplicity (omit distribution).